src/fileActions.jsx

```diff
@@ -436,7 +438,7 @@ const EditPermissionsModal = ({ selected, path }) => {
     );
 };
 
-const setDirectoryName = (val, setName, setNameError, setErrorMessage) => {
+const setInputName = (val, setName, setNameError, setErrorMessage, cwdInfo) => {
```

Comment:
minor nag: this change landed in the wrong commit.  The chosen name is also a bit weird, so if you're gonna move it to the other commit (which makes all the other changes to this function) maybe call it something like `setFileName()` instead?

Reply:
And it's a completely separate topic, but I dislike the structure of this function: I reworked it for my mkdir patch to be a simple function which took the proposed filename and returned a string describing an error (in case of error) or undefined for "no error".  I kept the state-modification logic in the component itself, which reduced the number of arguments to the error-checking function.